x-pack/plugins/fleet/server/services/epm/package_service.ts

```diff
@@ -60,14 +65,22 @@ export interface PackageClient {
     pkgVersion?: string;
     spaceId?: string;
     force?: boolean;
-  }): Promise<Installation | undefined>;
+  }): Promise<Installation>;
```

Comment:
Fleet team: Please let me know if you are happy with this change. I did not find a codepath for when this function could ever return `undefined`. Removing it makes this API a lot easier to consume but I'm happy to revert the change if there are concerns.

Reply:
Yes I think it make sense, and it seems there is no issue with the typescript check